.goreleaser.yml

```diff
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/goreleaser/goreleaser/v1.26.0/www/docs/static/schema-pro.json
```

Comment:
this is so VSCode shows the correct schema validation, based on the version of go releaser in Dagger, which I found to be 1.26